File: cpp/multiples-of-3-or-5/solution.cpp
// https://www.codewars.com/kata/514b92a657cdc65150000006

int solution(int until) 
{
    int sum = 0;
    for (int i = 1; i < until; i++) {
      if (i % 3 == 0 || i % 5 == 0) {
        sum += i;
      }
    }
    return sum;
}

File: csharp/multiples-of-3-or-5/solution.cs
// https://www.codewars.com/kata/514b92a657cdc65150000006

using System;
using System.Collections.Generic;
using System.Linq;

public static class Kata
{
  public static int Solution(int until)
  {
    if (until < 3) {
      return 0;
    }
    return Enumerable.Range(1, until-1)
      .Where(n => n % 3 == 0 || n % 5 == 0)
      .Sum();
  }
}

